src/CartModal.scss

```diff
@@ -0,0 +1,194 @@
+
+@import './theme/common';
+
+.cartmodal {
+
+  &__overlay {
+    position: fixed;
+    top: 0px;
+    right: 0px;
+    bottom: 0px;
+    left: 0px;
+    background: $mainTextColor;
+    opacity: 0;
+    visibility: hidden;
+    z-index: 2;
+    transition: all 0.3s ease 0s;
+  }
+
+  &__content {
+    position: fixed;
+    top: 0px;
+    bottom: 0px;
+    right: 0px;
+    overflow-y: scroll;
+    height: 100%;
+    width: 500px;
```

Comment:
500px width is fixed on mobile. Need to adjust this for mobile views so it wraps contents:
![image](https://user-images.githubusercontent.com/39800563/92155670-bce33a00-edf5-11ea-88f7-b4a5b89269b3.png)

Compared to ShopKit:
![image](https://user-images.githubusercontent.com/39800563/92155693-c66ca200-edf5-11ea-806e-4124393d67da.png)